src/JustSaying/Messaging/Interrogation/InterrogationResult.cs

```diff
@@ -20,5 +20,7 @@ public InterrogationResult(object data)
         /// <remarks>This property is intentionally untyped, because it is unstructured and subject to change.
         /// It should only be used for diagnostic purposes.</remarks>
         public object Data { get; }
+
+        internal static InterrogationResult Empty => new InterrogationResult(new {});
```

Comment:
If we do this, then it's the same shared one, rather than a new one on every use.

```suggestion
        internal static InterrogationResult Empty { get; } = new InterrogationResult(new {});
```

Reply:
Ah yes, my mistake 🙂